src/test/java/org/folio/processing/mapping/InstanceMappingTest.java

```diff
@@ -545,16 +545,26 @@ public void testMarcToInstanceWithRepeatableSubjects() throws IOException {
 
   @Test
   public void testMarcToInstanceWithRepeatableSubjectsButWithoutIndicators() throws IOException {
```

Comment:
@RomanChernetskyi  I hope there should be added a separate test for it. Because this one: "testMarcToInstanceWithRepeatableSubjectsButWithoutIndicators" was created for a separate bug. Please create a separate test for the current bug.

Reply:
After discussion with @RomanChernetskyi  it seems like it would be ok to combine these 2 cases. Resolving this comment.